Makefile

```diff
@@ -279,6 +279,13 @@ python/format:
 	black $(PYTHON_FILES)
 	@ echo "[$@]: Successfully formatted Python files!"
 
+# Run pytests, typically for unit tests.
+pytest/%: | guard/program/pytest
+pytest/%:
+	@ echo "[$@] Starting Python tests"
```

Comment:
I would suggest the echo statement include the `$*` reference to display what is being tested.

Reply:
ok